app/models/estateagentbusiness/RedressScheme.scala

```diff
@@ -74,14 +62,8 @@ object RedressScheme {
       {
         (__ \ "propertyRedressScheme").read[String].flatMap[RedressScheme] {
           case "01" => ThePropertyOmbudsman
-          case "02" => OmbudsmanServices
```

Comment:
We will need to be able to read the data in its existing state (with the deprecated answers) in play. This read will be used by the interrupt page.

Reply:
Readded in case objects alongside JsonReads for Other and OmbudsmanServices